python_modules/dagster/dagster/_core/definitions/run_status_sensor_definition.py

```diff
@@ -461,8 +468,17 @@ def inner(
             request_job=request_job,
             request_jobs=request_jobs,
         )
-        def _run_failure_sensor(context: RunStatusSensorContext):
-            return fn(context.for_run_failure())  # fmt: skip
+        @mirror_signature_of(fn)
```

Comment:
does `wraps` not cover this? 

https://docs.python.org/3/library/functools.html#functools.wraps

Reply:
so it does